index.html

```diff
@@ -78,7 +78,10 @@ <h1>Getting started</h1>
 &lt;article&gt;</code></pre>
             </figure>
           </p>
-          <p>The css stylesheet <code>Rapido.css</code>, you can download it at <strong>TODO</strong>.</p>
+          <p>The css stylesheet <code>rapido.css</code>, you can download it from <a href="https://raw.githubusercontent.com/nextbitlabs/Rapido/master/index.css">github</a> or directly use
```

Comment:
This sounds a bit too complicated, maybe `You can download the css stylesheet <code>rapido.css</code> from` ?

Reply:
It is a typo actually, it was solved in https://github.com/nextbitlabs/Rapido/pull/26. The update version is

> The css stylesheet rapido.css can be downloaded from github, in alternative you can simply use the following line within element <head>.